packages/flutter/test/widgets/custom_painter_test.dart

```diff
@@ -675,8 +675,8 @@ void _defineTests() {
       semantics: testSemantics2,
     )));
     expect(_PainterWithSemantics.shouldRebuildSemanticsCallCount, 2);
-    expect(_PainterWithSemantics.buildSemanticsCallCount, 2);
-    expect(_PainterWithSemantics.semanticsBuilderCallCount, 5);
+    expect(_PainterWithSemantics.buildSemanticsCallCount, 1);
```

Comment:
Because the object is now `const`, the `identical` test in `shouldRebuildSemantics` returns true in more cases.